src/nyc_trees/js/src/map.js

```diff
@@ -211,3 +213,57 @@ function getDomMapAttribute(dataAttName, domId) {
         value = $map.data(dataAttName);
     return value;
 }
+
+function startTrackingUserPosition(map) {
+    map.on('locationfound', onLocationFound, map);
+    map.on('locationerror', onLocationError, map);
+    map.on('unload', stopTrackingUserPosition, map);
+    map.locate({watch: true, enableHighAccuracy: true});
+}
+
+function stopTrackingUserPosition(map) {
+    var marker = map._userLocationMarker;
+    if (marker) {
+        map.removeLayer(marker);
+    }
+    map.off('locationfound', onLocationFound);
+    map.off('locationerror', onLocationError);
+    map.off('unload', stopTrackingUserPosition);
+    map.stopLocate();
+}
+
+function onLocationFound(e) {
+    var map = this,
+        marker = map._userLocationMarker,
+        latlng = e.latlng;
+
+    // For testing, move points near Azavea to southern Manhattan
+    var azaveaLatLng = L.latLng(39.9583208, -75.1585257),
+        manhattanLatLng = L.latLng(40.7030809, -74.0129269);
```

Comment:
This PR can't be tested or demonstrated from Azavea without that code. It should be safe to leave it in, and we can also ask Parks to test it from staging and then remove it.